geonode/base/templates/base/_resourcebase_snippet.html

```diff
@@ -37,14 +37,14 @@ <h4>
               </h4>
             </div>
             <div class="col-xs-2">
-              <h4>
+              <h4>{% endverbatim %}
```

Comment:
You first endverbatim and than verbatim? Is that correct? It does seem to work in the UI though.

Reply:
I have tested all my changes extensively. Without ending the "verbatim" here, the translation at this point will not work otherwise.

Reply:
@afabiani ? I would approve this.